internal/util/unregisterer.go

```diff
@@ -18,6 +18,22 @@ func WrapWithUnregisterer(reg prometheus.Registerer) *Unregisterer {
 	}
 }
 
+func isUncheckedCollector(c prometheus.Collector) bool {
```

Comment:
Is that the official definition of the unchecked collector? Could you add a comment?

Reply:
I added a comment to clarify this further.